docs/concepts.rst

```diff
@@ -103,3 +103,46 @@ See also
 
 * :term:`Kinto glossary <CRUD>`, for formal definitions of terms used in the documentation
 * :blog:`Our blog post <en/handling-permissions>` on permissions design in *Kinto*
+
+
+.. _technical-architecture:
+
+Technical Architecture
+======================
+
+Every concept of bucket, group, collection or record leverages the REST *resource*
+notion of *Cliquet*.
+
+*Cliquet* is a toolkit which brings abstractions and
+good practices to build HTTP microservices, such as data-driven REST APIs.
+
+*Cliquet* is relies on *Cornice* and *Pyramid* to implement the endpoints
+and HTTP request/response handling.
+
+.. image:: images/architecture.svg
```

Comment:
The comments for the sequence diagram are only visible in the source. It is just to be able to edit the sequence diagram on the website. For architecture, it's an SVG so we can edit it directly in Inkscape